api/.env.example

```diff
@@ -1,15 +1,18 @@
-DB_URL=postgres://postgres:postgress@localhost:5432/postgres
+DB_URL=postgres://postgres:postgres@localhost:5432/postgres
 NODE_ENV=development
 PORT=8080
 JWT_SECRET=''
 ALBY_API_TOKEN="dkkfnfnf"
 ALBY_REDIRECT_URL="http://localhost:8080"
 ALBY_CLIENT_SECRET="secret"
-ALBY_SECRET_ID="secret"
+ALBY_CLIENT_ID="secret"
 ALBY_API_URL="https://api.alby.com"
+# We are using an LND node on voltage.cloud for staging and production
```

Comment:
We shouldn't need to include comments in .env.sample

Reply:
You are right, I'll try to include those information in the README instead.